src/api/stats/matic/getQuickLpApys.js

```diff
@@ -14,63 +14,16 @@ const oracleId = 'QUICK';
-const quickEthRewardPool = '0xD1C762861AAe85dF2e586a668A793AAfF820932b'; // Quick-ETH
-const celEthRewardPool = '0x8917692e0Bdb47AF1D36837805E141Ed79065dFC'; // Cel-ETH
-const wiseEthRewardPool = '0xb11856d3Aea0203e50B8520479C6332daBcF3f82'; // WISE-ETH
-const defi5EthRewardPool = '0xf563fAe71bDAcDD370098CeCff14dbe2c9518a6b'; // Defi5-ETH
-const ubtEthRewardPool = '0x219670F92CC0e0ef1C16BDB0aE266F0472930906'; // UBT-ETH
-
-// TODO rewrite it, put the rewardPool address inside quickLpPools.json
 const getQuickLpApys = async () => {
-  let poolMaticQuick = pools.filter(pool => pool.name === 'quick-quick-matic')[0];
-  let poolEthMatic = pools.filter(pool => pool.name === 'quick-eth-matic')[0];
-  let poolEthBtc = pools.filter(pool => pool.name === 'quick-eth-btc')[0];
-  let poolEthUsdt = pools.filter(pool => pool.name === 'quick-eth-usdt')[0];
-  let poolUsdcUsdt = pools.filter(pool => pool.name === 'quick-usdc-usdt')[0];
-  let poolLinkEth = pools.filter(pool => pool.name === 'quick-link-eth')[0];
-  let poolAaveEth = pools.filter(pool => pool.name === 'quick-aave-eth')[0];
-  let poolmaUsdcUsdc = pools.filter(pool => pool.name === 'quick-mausdc-usdc')[0];
-  let poolQuickEth = pools.filter(pool => pool.name === 'quick-quick-eth')[0];
-  let poolCelEth = pools.filter(pool => pool.name === 'quick-cel-eth')[0];
-  let poolWiseEth = pools.filter(pool => pool.name === 'quick-wise-eth')[0];
-  let poolDefi5Eth = pools.filter(pool => pool.name === 'quick-defi5-eth')[0];
-  let poolUbtEth = pools.filter(pool => pool.name === 'quick-ubt-eth')[0];
-
   let apys = {};
-  let apy = await getPoolApy(maticQuickRewardPool, poolMaticQuick, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(ethMaticRewardPool, poolEthMatic, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(ethBtcRewardPool, poolEthBtc, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(ethUsdtRewardPool, poolEthUsdt, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(usdcUsdtRewardPool, poolUsdcUsdt, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(linkEthRewardPool, poolLinkEth, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(aaveEthRewardPool, poolAaveEth, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(maUsdcUsdcRewardPool, poolmaUsdcUsdc, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(quickEthRewardPool, poolQuickEth, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(celEthRewardPool, poolCelEth, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(wiseEthRewardPool, poolWiseEth, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(defi5EthRewardPool, poolDefi5Eth, 137);
-  apys = { ...apys, ...apy };
-  apy = await getPoolApy(ubtEthRewardPool, poolUbtEth, 137);
-  apys = { ...apys, ...apy };
+
+  let promises = [];
+  pools.forEach(pool => promises.push(getPoolApy(pool.rewardPool, pool, 137)));
+  const values = await Promise.all(promises);
```

Comment:
this will be rate-limited?
i think we need this non-parallel before we get private rpc